policy/release/olm.rego

```diff
@@ -122,6 +122,34 @@ deny contains result if {
 	result := lib.result_helper(rego.metadata.chain(), [error])
 }
 
+# METADATA
+# title: Unpinned images in input snapshot
+# description: >-
+#   Check the input snapshot for the presence of unpinned image references.
```

Comment:
ec-cli should always resolve image references to a digest. Is that not happening?